src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -631,7 +623,7 @@ def get_job_logs(
         except Exception as e:
             raise transform_job_exception(e)
 
-    def cancel_job(self, job_id: str, parent_job_id: str = None) -> None:
+    def cancel_job(self, job_id: str, parent_job_id: str = None) -> bool:
```

Comment:
Facilitate testing by returning a boolean here, so we know whether, based on existing data, it has already been cancelled.